internal/csi-common/controllerserver-default.go

```diff
@@ -94,3 +94,11 @@ func (cs *DefaultControllerServer) ControllerGetVolume(
 ) (*csi.ControllerGetVolumeResponse, error) {
 	return nil, status.Error(codes.Unimplemented, "")
 }
+
+// ControllerModifyVolume modifies existing volume.
+func (cs *DefaultControllerServer) ControllerModifyVolume(
```

Comment:
this is so ugly... Do you know why `DefaultControllerServer` does not extend/contain the `csi.UnimplementedControllerServer` so that all these functions can be removed?

Reply:
Good catch! this was missing in two places, add it and also removed unwanted methods as well.